Client/my-story-illustrator-client/src/header/header.js

```diff
@@ -0,0 +1,35 @@
+import GoBackButton from "./go_back_button";
+import UserSettingsButton from "./user_settings_button";
+
+function Header(props){
```

Comment:
This is a pretty good start. Just remember to use chakra components rather than plain HTML components as much as you possible can, and your life will be easier. Here's my advice for switching to chakra ui components.

- instead of header, let's use Flex. This will automatically give you a horizontal flex container for the header. 
- I would wrap lines 21-28 in a Center component. This is an easy way to make sure it stays centered on the screen.
- On line 22, instead of form, a Box or Container would work.
- instead of input, use the Chakra element, Input. It comes with some styles that look pretty good on their own.
- instead of h1, use the Chakra Heading element.

Read the Chakra UI documentation if you are unsure how to use something. If we do this, it will mostly style itself.